### ISSUE TITLE
Hats/Helmets/Overlays not rendering as 3D

### ISSUE THREAD

**Issue by StormboundProject** [crafatar/crafatar#259]
Are helmets/hats/overlays supposed to render as if they were "pasted" onto the head itself and not slightly outside of it? I've seen issue #25 but it doesn't seem like anything was done on this feature specifically. It would be great if there was an option for overlays to render realistically though!

**Comment by jomo** [comment]
I agree it would look better with the overlay rendered as it does ingame.

Accepting PRs!
